src/App.jsx

```diff
@@ -1,9 +1,11 @@
-function App() {
+import ButtonComponent from "./components/exercise4/buttonComponent";
 
+function App() {
   return (
     <>
+      <ButtonComponent />
     </>
-  )
+  );
```

Comment:
Since you are returning only one component, I think there is no need for those tags because it's not helping in any thing. You can probably check for it and see.

Reply:
Thank you and am working on it right away

Reply:
Now it is resolved